types/types.hh

```diff
@@ -832,6 +832,12 @@ to_bytes(bytes_view x) {
     return bytes(x.begin(), x.size());
 }
 
+inline
+bytes
+to_bytes(std::string_view x) {
+    return to_bytes(to_bytes_view(x));
```

Comment:
Wow, the definition of to_bytes_view(string_view) is in a different header file (bytes.hh)? What a mess :-(